moveit_pr2/package.xml

```diff
@@ -14,9 +15,8 @@
 
   <buildtool_depend>catkin</buildtool_depend>
 
-  <run_depend>pr2_moveit_plugins</run_depend>
-  <run_depend>pr2_moveit_config</run_depend>
-  <run_depend>pr2_moveit_tutorials</run_depend>
+  <exec_depend>pr2_moveit_plugins</exec_depend>
+  <exec_depend>pr2_moveit_config</exec_depend>
```

Comment:
Alphabetical order of the entries, if we are too concerned.

Reply:
Resolved